elements/body.html

```diff
@@ -1,20 +1,14 @@
 <article class="article">
   <header class="article-header">
     <h2>
-      <code>
-        <!-- Actual Tag Name  -->
-        &lt;body>
-      </code>
-      <!-- Audio -->
+      <code>&lt;body&gt;</code>
       <audio src="audio/body.mp3" controls></audio>
```

Comment:
body, header, footer, a, and br all have audio recordings (those are the 5 elements that the yweb group completed).